includes/derivatives.inc

```diff
@@ -151,7 +152,18 @@ function islandora_pdf_add_pdfa_derivative(AbstractObject $object, $force = FALS
   $executable = variable_get('islandora_pdf_path_to_gs', '/usr/bin/gs');
   $temp = file_create_filename('pdfa.pdf', 'temporary://');
   $derivative_file_uri = drupal_realpath($temp);
-  $command = "$executable -dPDFA -dNOOUTERSAVE -dUseCIEColor -sProcessColorModel=DeviceRGB -sDEVICE=pdfwrite -o $derivative_file_uri -dPDFACompatibilityPolicy=1 $source ";
+
+  // XXX. As of Ghostscript version 9.11, the use of -dUseCIEColor is not
+  // recommended.
+  // see https://ghostscript.com/pipermail/gs-devel/2014-July/009693.html.
+  $use_ci_color = "-dUseCIEColor";
+  $version_command = "$executable --version";
+  exec($version_command, $version);
+
+  if ($version[0] >= $min_version) {
```

Comment:
Should probably do a more sophisticated version comparison here, in case the version happens to return a point release something like this: `9.11.2`. In this case it won't cast to a number... and then you are doing `>=` comparison on strings...

Maybe use something like this instead: 
https://php.net/version_compare

Reply:
Eliminated the need for this check by making this an admin option